Uplift/Controllers/HomeViewController.swift

```diff
@@ -89,17 +89,18 @@ class HomeViewController: UIViewController {
 
             // Reload Today's Classes section
             self.collectionView.reloadSections(IndexSet(integer: 1))
-
             self.decrementNumPendingNetworkRequests()
         })
 
+//MARK: Networking for .LookingForCategories
```

Comment:
Can we remove this commented code? If not, can we document why we need it and how we will use it in the future?

Reply:
Will add documentation to next PR